.github/workflows/test-cleardown.yaml

```diff
@@ -1,14 +1,19 @@
 name: Test Cleardown Workflow
+# Intended to run if
+# actor is not the queue bot and
+# the branch deleted is either
+# a task branch or
+# a dependabot branch
 
 on:
   delete:
     branches:
-      - main
       - 'task/*'
+      - dependabot/*
 
 jobs:
   metadata:
-    if: github.event.ref_type == 'branch' && startswith(github.event.ref, 'task/') && github.actor != 'github-merge-queue[bot]'
+    if: ( ${{ github.event.ref_type == 'branch' && github.actor != 'github-merge-queue[bot]' }} )
```

Comment:
Do we no longer care if the branch is task/* or dependabot/* ?

Reply:
This job will only run on the deletion of task/* & dependabot/* (limes 8 - 12) so we don't need to highlight this again in the if statement.